syft/format/cyclonedxjson/decoder.go

```diff
@@ -75,7 +74,7 @@ func (d decoder) Identify(reader io.ReadSeeker) (sbom.FormatID, string) {
 		return "", ""
 	}
 
-	id, version := getFormatInfo(doc.JSONSchema, doc.BOMFormat, doc.SpecVersion)
```

Comment:
is json-schema optional?

Reply:
`$schema` is not valid for CycloneDX 1.2, at least: https://github.com/CycloneDX/specification/blob/1.2/schema/bom-1.2.schema.json